src/nvim/ui.coffee

```diff
@@ -73,15 +74,31 @@ class UI extends EventEmitter
     @cursor.style.height = @char_height + 'px'
 
     if config.blink_cursor
-      @cursor.classList.add('blink')
+      @startCursorBlink()
     else
-      @cursor.classList.remove('blink')
+      @stopCursorBlink()
+
+  startCursorBlink: ->
+    if config.blink_cursor
+      @cursor.classList.add 'blink'
+
+  stopCursorBlink: ->
+    @cursor.classList.remove 'blink'
 
   init_event_handlers: ->
+    resumeBlink = _.debounce(@startCursorBlink.bind(this), 250)
```

Comment:
Fat arrows? This is my first time using coffeescript for anything. xD How would you change this?